src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -685,22 +685,21 @@
 helps['role assignment create'] = """
 type: command
 short-summary: Create a new role assignment for a user, group, or service principal.
+long-summary: In a future release, --scope argument will become required for creating a role assignment. Please explicitly specify --scope.
 examples:
-  - name: Create role assignment for an assignee.
-    text: az role assignment create --assignee sp_name --role a_role
+  - name: Create role assignment to grant an assignee Reader role on a virtual machine.
```

Comment:
I find this description cumbersome. @dcaro, @dbradish-microsoft, any though on how we can improve it?

Reply:
```suggestion
  - name: Create role assignment to grant Reader role to the specified assignee on an Azure virtual machine
```
I am not sure if we have any guidelines for the length of this field so I tried to keep is short. We can be more explicit if we have more space.

Reply:
```suggestion
  - name: Create role assignment to grant the specified assignee the Reader role on an Azure virtual machine.
```